front/pages/poke/[wId]/data_sources/[name]/index.tsx

```diff
@@ -226,6 +231,16 @@ export const getServerSideProps = withSuperUserAuthRequirements<{
     }
   }
 
+  // Getting groups that a Slack bot/workflow can be assigned to
+  const authForSlackBot = await Authenticator.internalAdminForWorkspace(
+    owner.sId
+  );
+  const groupsForSlackBot = (
+    await GroupResource.listWorkspaceGroups(authForSlackBot)
+  )
+    .filter((g) => g.kind !== "system")
```

Comment:
Let's add a `isSystem` method on the `GroupResource`.

Reply:
👍